content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html

```diff
@@ -0,0 +1,34 @@
+---
+title: برنامج تعليمي تفاعلي - إنشاء عنقود
+weight: 20
+headless: true
+toc_hide: true
+_build:
+  list: never
+  publishResources: false
+---
```

Comment:
IMO, this file does not need to exist in any new localization.

Reply:
Thanks team ! do you mean that it is not required as min or that it should even been translated

Reply:
I would leave this file out entirely. If we could, we'd serve a 410 Gone error for existing localizations; new localizations wouldn't need any error page.

Reply:
same for this on exists under the Korean l10ncan we just keep it as it is tiny translation ? https://github.com/kubernetes/website/blob/main/content/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html